src/js/stores/AppsStore.js

```diff
@@ -147,29 +148,47 @@ function processApp(app) {
   return app;
 }
 
-function processEmptyGroup(group) {
-  return Object.assign(processApp({id: group.id}), {
-    isGroup: true,
-    status: null
+function addData(parent, child) {
+  Object.keys(parent).forEach(key => {
+    if (key === "health") {
+      parent[key].forEach((value, index) => {
+        value.quantity += child[key][index].quantity;
```

Comment:
I'd suggest to be defensive here. We should check `child[key] != null && child[key]][index] != null`.